src/network/networkctl.c

```diff
@@ -1405,6 +1415,35 @@ static int link_status_one(
                 if (r < 0)
                         return table_log_add_error(r);
 
+        } else if (streq_ptr(info->netdev_kind, "bond")) {
+                static const struct {
+                        const char *mode;
+                } mode_table[] = {
+                        { "balance-rr" },
+                        { "active-backup" },
+                        { "balance-xor" },
+                        { "broadcast" },
+                        { "802.3ad" },
+                        { "balance-tlb" },
+                        { "balance-alb" },
+                };
```

Comment:
Maybe, create shared/bond-util.[ch] and expose `bond_mode_{to,from}_string()` in netdev/bond.c, then use it here.

Reply:
I thought of that wont's that be an overkill ?

Reply:
I thought then the functions can be used in network-generator. But OK. Let's merge this as it is, and do that in the future.

Reply:
Then it would be useful. sure.